pkg/utils/report/results.go

```diff
@@ -159,6 +159,26 @@ func ToPolicyReportResult(pol engineapi.GenericPolicy, ruleResult engineapi.Rule
 		addPodSecurityProperties(pss, &result)
 	}
 
+	if pol.AsMutatingAdmissionPolicy() != nil {
+		result.Source = "MutatingAdmissionPolicy"
+		if binding := ruleResult.MutatingAdmissionPolicyBinding(); binding != nil {
+			addProperty("binding", binding.Name, &result)
+		}
+	}
+
+	if pol.AsValidatingAdmissionPolicy() != nil {
+		result.Source = SourceValidatingAdmissionPolicy
+		result.Policy = ruleResult.Name()
+		if ruleResult.ValidatingAdmissionPolicyBinding() != nil {
+			addProperty("binding", ruleResult.ValidatingAdmissionPolicyBinding().Name, &result)
+		}
+	}
+	if pol.AsValidatingPolicy() != nil {
+		result.Source = SourceValidatingPolicy
+	}
+	if pol.AsImageValidatingPolicy() != nil {
+		result.Source = SourceImageValidatingPolicy
+	}
```

Comment:
This is already done [here](https://github.com/kyverno/kyverno/blob/ada597abf4915e3f45277de36ce48dfd48e2e808/pkg/utils/report/results.go#L117-L139). You just need to add one for MAPs.